include/seqan3/core/type_traits/template_inspection.hpp

```diff
@@ -57,17 +57,17 @@ struct transfer_template_args_onto
  */
 template <template <typename ...> typename source_template,
           template <typename ...> typename target_template,
-          typename ... source_arg_types>
+          typename ...source_arg_types>
 //!\cond
     requires requires ()
     {
-        std::declval<target_template<source_arg_types...>>();
+        std::declval<target_template <source_arg_types...>>();
```

Comment:
Are we sure about this?

Reply:
nope this is a regex error. thanks for catching this